app/styles/mixins.scss

```diff
@@ -25,6 +25,11 @@
   -moz-transition: $string;
 }
 
+@mixin animation($string) {
+  animation: $string;
+  -webkit-animation: $string;
```

Comment:
Is there a reason you're only including `-webkit-`? We needz browser supportz, right?

Yes, I looked at the scss :+1: